vanessa-runner/Dockerfile

```
@@ -1,5 +1,5 @@
 ARG DOCKER_REGISTRY_URL
-FROM ${DOCKER_REGISTRY_URL}/oscript:1.0.21
+FROM ${DOCKER_REGISTRY_URL:+"$DOCKER_REGISTRY_URL/"}oscript:1.0.21
```

Hadolint (2.14.0)

[error] unexpected ':'
expecting '@', '\', a new line followed by the next instruction, or the image tag

(DL1000)